apps/pie-docs/src/components/progress-stepper/progress-stepper.md

```diff
@@ -0,0 +1,67 @@
+---
+eleventyNavigation:
+    key: 'Progress Stepper'
+    parent: Components
+---
+
+## Overview
+The purpose of progress steppers is to guide users through a series of steps, visually indicating their progress and completion status. It helps users navigate complex processes, maintain a sense of orientation, and ensures a smoother user journey.
+
+Progress steppers are commonly used in multi-step forms, onboarding processes, checkout flows or any situation where a complex task is divided into manageable steps. They enhance the user experience by providing a visual roadmap and aiding in task completion.
```

Comment:
I think using the words "continuous" and "discrete" might be a good way of distinguishing the progress-bar from the progress-stepper. What do you think?